app/src/main/res/layout/item_status_detailed.xml

```diff
@@ -1,10 +1,13 @@
 <?xml version="1.0" encoding="utf-8"?>
 <RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:sparkbutton="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
     android:id="@+id/status_container"
     android:layout_width="match_parent"
     android:layout_height="wrap_content"
+    android:clipChildren="false"
```

Comment:
It still clips animation for me...

Reply:
Okay, I see, before it was clipped on top and not it's clipped only at the bottom. Weird that it still does that but it's better ofc

Reply:
I guess we would need to set it on the containing view via code if we want no clipping at all

Reply:
I think it's good as-is, we can improve it later